deployment/v3/mosip/digitalcard/copy_cm_func.sh

```
@@ -26,3 +26,7 @@ else
    kubectl -n $4 delete --ignore-not-found=true $RESOURCE $2
    kubectl -n $3 get $RESOURCE $2 -o yaml | sed "s/namespace: $3/namespace: $4/g" |  kubectl -n $4 create -f -
 fi
+<<<<<<< HEAD
```

Shellcheck (0.11.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)